src/SourceCode.Chasm.Tests/TreeNodeListTests.cs

```diff
@@ -25,38 +26,41 @@ public static class TreeNodeListTests
 
         #region Methods
 
+        private static void AssertEmpty(TreeNodeList treeNodeList)
```

Comment:
This looked like duplicate code I added in a helper method.

Reply:
Good idea, thanks